pandas_gbq/gbq.py

```diff
@@ -15,6 +15,8 @@
 from pandas.core.common import PandasError
 from pandas.compat import lzip, bytes_to_str
 
+logger = logging.getLogger(__name__)
```

Comment:
Generally the logging settings should be at the application level, rather than set by the code - so set at pytest. Though I don't think setting a `NullHandler` will interfere with anyone's settings